src/the_way/gist.rs

```diff
@@ -1,25 +1,45 @@
-//! Code related to syncing snippets to Gist
+//! Code related to dealing with Gists
 use std::collections::HashMap;
 
 use color_eyre::Help;
 
 use crate::errors::LostTheWay;
 use crate::gist::{CreateGistPayload, GistClient, GistContent, UpdateGistPayload};
-use crate::the_way::TheWay;
+use crate::the_way::{snippet::Snippet, TheWay};
 use crate::utils;
 
 /// Gist description
 const DESCRIPTION: &str = "The Way Code Snippets";
 /// Heading for the index.md file
 const INDEX: &str = "# Is it not written...\n";
-const USER_AGENT: &str = "the-way";
 
 impl TheWay {
+    /// Import Snippets from a regular Gist
+    pub(crate) fn import_gist(&mut self, gist_url: &str) -> color_eyre::Result<Vec<Snippet>> {
+        // it's assumed that access token is not required when reading Gists
+        let client = GistClient::new(None)?;
```

Comment:
You'd still need the access token for secret Gists right?

Reply:
See my comment above https://github.com/out-of-cheese-error/the-way/pull/79#discussion_r503505527.